### PR TITLE
[screensaver.fTVscreensaver] 1.0.0

### PR DESCRIPTION
**Pull request by sualfred**
### Description

Reworked my old screensaver from a few years back to make it comaptible to Leia + Matrix
Provides a fTV like slideshow.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Rechi** [comment]
I've seen you used `script.screensaver.ftvscreensaver` previously as add-on id.
I don't mind if you want to use `screensaver.fTVscreensaver`, but there are still some reverences to the old id in your submission and the add-on won't auto update.

**Comment by sualfred** [comment]
@Rechi 
Just renamed it because all other screensavers are not using 'script.screensaver.bla'
Changed the missing stuff

File: screensaver.fTVscreensaver/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+0.1.2
```
Comment (enen92) [inline screensaver.fTVscreensaver/changelog.txt]: Please try to use only the news element in the future

File: screensaver.fTVscreensaver/screensaver.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        for i in range(1,17):
+            winprop('fTVscreensaver.Fanart.%s' % i, clear=True)
+        self.close()
+
+
+if __name__ == '__main__':
+    screensaver = Screensaver('screensaver.fTVscreensaver.xml',ADDON_PATH,'default')
+    screensaver.doModal()
+    del screensaver
+    sys.modules.clear()
```
Comment (enen92) [inline screensaver.fTVscreensaver/screensaver.py]: any specific reason to clear the sys.modules when finishing the addon?

Comment (sualfred) [reply screensaver.fTVscreensaver/screensaver.py]: To be honest, It's a leftover from back in the days where I had no clue about Python and I just made this script based on another one.
But yeah, makes no sense. I'll remove it.

File: screensaver.fTVscreensaver/changelog.txt (hunk 1 above)
Comment (sualfred) [reply screensaver.fTVscreensaver/changelog.txt]: Leftover file. Missed it. Will be removed.
There are reasons why I don't use the news element: Identation. It's a pita to format the text correctly. And a changelog.txt is more useful by looking at source code on GitHub or anywhere else. I wish Kodi never had removed the native changelog.txt support.

**Pull request merged**